homeassistant/components/vacuum/roomba.py

```diff
@@ -5,17 +5,16 @@
 https://home-assistant.io/components/vacuum.roomba/
 """
 import asyncio
+import homeassistant.helpers.config_validation as cv
```

Comment:
This line should go below the lines with `homeassistant.const`. Sorry for not being clear.